zeppelin-web/src/app/notebook/paragraph/result/result.controller.js

```diff
@@ -326,41 +326,50 @@ function ResultCtrl ($scope, $rootScope, $route, $window, $routeParams, $locatio
   /**
    * Render multiple sub results for custom display
    */
-  $scope.renderCustomDisplay = function (type, data) {
-    // get result from intp
+  $scope.renderCustomDisplay = function(type, data) {
+
+    var splits = data.split(' ');
+    type = data.substring(0, splits[0].length)
+    data = data.substring(splits[0].length, data.length)
+
     if (!heliumService.getSpellByMagic(type)) {
```

Comment:
What i mean is, if text (not paragraph) includes `%fancytext`, it will throw an error like 

> `%fancytext` display doesn't exist. 

In this case, user doesn't want to use `%fancytext` display and it doesn't exist. Just text includes that kind of text. 

LINE 335.